pine/src/main/java/com/distkv/pine/api/Pine.java

```diff
@@ -3,18 +3,31 @@
 import com.distkv.pine.components.PineTopper;
 import com.distkv.pine.runtime.PineRuntime;
 
+/**
+ * The API class of Pine.
+ */
 public class Pine {
 
+  /// The Pine runtime.
   private static PineRuntime runtime;
 
-  public static void init() {
+  /**
+   * Initial the Pine.
+   *
+   * @param address The address of distkv server to connect.
+   */
+  public static void init(String address) {
     runtime = new PineRuntime();
-    runtime.init();
+    runtime.init(address);
   }
 
+  /**
+   * Shutdown the Pine.
+   */
   public static void shutdown() {
```

Comment:
how about use jdk8 closeable interface to release the resources, so we can use the `try-with-resource` feture of jdk8. which make code clear

Reply:
I'm wondering if it's easy to use try-with on a static method. Could you confirm that?

Reply:
yes, can't.